isb_web/manage.py

```diff
@@ -225,6 +229,8 @@ async def auth(request: starlette.requests.Request):
         # if login is for annotation purpose, add access token as query param
         redirect_url += "?access_token=" + token["access_token"]
         return starlette.responses.RedirectResponse(url=redirect_url)
+    elif "thing" in request.query_params:
+        return starlette.responses.RedirectResponse(url=f"/thingpage/{request.query_params.get('thing')}")
```

Comment:
New support to bounce back over to the thing page when redirecting back from orcid login.